add-domain

```diff
@@ -1,3 +1,5 @@
+25a40529.pws-assistenz.de
```

Comment:
Interesting as this domain are involving `sedoparking.com` (https://github.com/mypdns/matrix/issues/1187)